test/TestRunner.py

```diff
@@ -20,8 +20,7 @@
 import os
 
 import pytest
-
-from ansiblelint import formatters
+import ansiblelint.formatters as formatters
```

Comment:
Keep the separation between project imports and third parties.
```suggestion

import ansiblelint.formatters as formatters
```